dropwizard-testing/src/main/java/io/dropwizard/testing/junit/DropwizardAppRule.java

```diff
@@ -21,12 +22,51 @@
  * override the {@link #newApplication()} method to provide your application instance(s).
  * </p>
  *
+ * <p>
+ * Using DropwizardAppRule at the suite level can speed up test runs, as the application is only started and stopped
+ * once for the entire suite:
+ * </p>
+ *
+ * <pre>
+ * @RunWith(Suite.class)
```

Comment:
I believe, the `@` symbol need to be reprseneted like HTML number `&#064;` or be wrapped by the [@literal](http://docs.oracle.com/javase/1.5.0/docs/tooldocs/solaris/javadoc.html#{@literal}) tag. Otherwise, the `javadoc` tool interpretes `@RunWith` like a Java annotation.